test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -492,23 +492,23 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
         It "Fails to import incompatible module if implicit WinCompat is disabled in config" {
             $LogPath = Join-Path $TestDrive (New-Guid).ToString()
             $ConfigPath = Join-Path $TestDrive 'powershell.config.json'
-            '{"DisableImplicitWinCompat" : "True"}' | Out-File -Force $ConfigPath
+            '{"DisableImplicitWinCompat" : true}' | Out-File -Force $ConfigPath
```

Comment:
This looks like a mistake previously where it should have always been `true` and not the string `"True"`.  I think this is the only boolean setting in the config file and is not set by default, so it's unlikely to impact users.